cni/network/network.go

```diff
@@ -349,6 +367,18 @@
 	iptables.DisableIPTableLock = nwCfg.DisableIPTableLock
 	plugin.setCNIReportDetails(nwCfg, CNI_ADD, "")
 
+	// acquire cni lock file
+	config := &common.PluginConfig{}
+	acquireLockForStore(config, plugin)
+	defer releaseLockForStore(plugin)
```

Comment:
>defer releaseLockForStore(plugin) [](http://example.com/codeflow?start=1&length=33)

Shouldnt this be above acquire, always safe to release in case it got acquired above and failed after

Reply:
nope not needed, if this fails no need to unlock